core/web/loader/node.go

```diff
@@ -20,15 +20,15 @@ func (b *nodeBatcher) loadByChainIDs(ctx context.Context, keys dataloader.Keys)
 	keyOrder := make(map[string]int, len(keys))
 	// Collect the keys to search for
 	// note backward compatibility -- this only ever supported evm chains
-	var evmrelayIdStrs []string
+	evmrelayIDs := make([]relay.ID, 0)
```

Comment:
Why `make` with zero `len`? Did you mean to set `cap` to `len(keys)` too?
```suggestion
	evmrelayIDs := make([]relay.ID, 0, len(keys))
```

Reply:
laziness i suppose. :) . changed now